code/modules/jobs/job_types/research_director.dm

```diff
@@ -51,6 +51,8 @@
 	backpack = /obj/item/storage/backpack/science
 	satchel = /obj/item/storage/backpack/satchel/tox
 
+	skillchips = list(/obj/item/skillchip/job/research_director)
```

Comment:
we really ought to give RDs that roboticist skillchip that tells you which cyborg wire is which

Reply:
Yes, it would be out of scope. But it's an easy thing to do?